README.md

```diff
@@ -71,6 +71,17 @@ For running in CI servers, we use Karma which works perfectly fine!
 
     - `entry` should be set to the test entry file. For example, `./test-entry.js`.
 
+    - optionally set `webpackMiddleware` which will be merged with some default options when we initialize webpack-middleware. This can be useful in case you need to enable polling:
+    ```
+      webpackMiddleware: {
```

Comment:
I chose webpackDevMiddleware to be consistent with karma.conf

I do think it's strange to include it in webpack config, since it's not *really* a webpack option but I guess it's not hurting anyone.